eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -0,0 +1,100 @@
+.\" Manpage for eos-updater-flatpak-installer.
+.\" Documentation is under the same licence as the eos-updater package.
+.TH man 8 "13 Apr 2017" "1.0" "eos\-updater\-flatpak\-installer man page"
+.\"
+.SH NAME
+.IX Header "NAME"
+eos\-updater\-flatpak\-installer — Endless OS Updater - Flatpak Installer
+.\"
+.SH SYNOPSIS
+.IX Header "SYNOPSIS"
+.\"
+\fBeos\-updater\-flatpak\-installer [\-m \-\-mode\fB] [\-p \-\-pull\fB]
+.\"
+.SH DESCRIPTION
+.IX Header "DESCRIPTION"
+.\"
+\fBeos\-updater\-flatpak\-installer\fP installs downloaded flatpaks on Endless OS
+updates upon rebooting into the new update. It is the part of the system that
+ensures that when new flatpaks are installed on OS Updates, they are
+installed "seamlessly", only appearing when rebooting into the new system.
+.PP
+.SH OPTIONS
+.IX Header "OPTIONS"
+.\"
+.IP "\fB\-m\fP, \fB\-\-mode\fP"
+Which mode to run the flatpak installer in. When set to \fBperform\fP, the
+installer will examine the list of applications to install in
+$(datadir)/eos-application-tools/flatpak-autoinstall.d/*,
+/var/lib/eos-updater/flatpak-autoinstall.d/override/* and the counter state
+in /var/lib/eos-application-tools/flatpak-autoinstall.progress, then for each
+basename, perform only newly updated actions and save the most up to date
+serial number for each file in
+\".
+When set to \fBstamp\fP, \fBeos\-updater\-flatpak\-installer\fP will only save
+the updated actions to
+/var/lib/eos-application-tools/flatpak-autoinstall.progress. When set to
+\fBcheck\fP, the tool will check to see if all actions are applied (e.g., that
+applications that should have been installed are installed and every app that
+should have been removed are removed).
+.\"
+.IP "\fB\-p\fP, \fB\-\-pull\fP"
+Whether to pull flatpaks as well as deploying them. By default, flatpaks are
+not pulled during this step, it is expected that they will be pulled by
+\fBeos\-updater\fP(8) when it fetches and applies system updates.
+.\"
+.SH "KERNEL COMMAND LINE"
+.IX Header "KERNEL COMMAND LINE"
+.\"
+If the \fPeos\-updater\-flatpak\-installer.service\fP unit is started when
+the kernel command line contains \fPeos\-updater\-disable\fP, then 
+it will have no effect.
+.\"
+.SH "EXIT STATUS"
+.IX Header "EXIT STATUS"
+.\"
+\fBeos\-updater\-flatpak\-installer\fP returns \fBEXIT_SUCCESS\fP in case of normal
+operation and \fBEXIT_FAILURE\fP in case a file was missing or we were unable
+apply some actions. It also returns \fBEXIT_FAILURE\fP when \fB\-\-mode\fP is
+set to \fBcheck\fP and flatpaks expected to be installed or removed are
+not installed or removed.
```

Comment:
It would be good if the exit status could be used to differentiate between failure due to invalid arguments, and failure due to the system state not matching what `check` expects.

See `eos-update-server.8` for an example of this kind of thing. Listing the exit statuses as an itemised list makes it a bit easier to read.